AppCenter/AppCenterTests/MSHttpClientTests.m

```diff
@@ -52,15 +54,42 @@ - (void)tearDown {
   [self.reachabilityMock stopMocking];
 }
 
-- (void)testCreateInitWithMaxConnections {
```

Comment:
In the `initWithMaxConnections` now contains compressionEnabled. The test title should reflect that and the compressionEnabled should be tested.